### ISSUE TITLE
Unit tests fail locally using `main` branch

### ISSUE THREAD

**Issue by fluiddot** [testing-library/jest-native#75]
- `react-native` or `expo`: react-native
- `@testing-library/react-native` version: 6.0.0
- `jest-preset`: N/A
- `react-native` version: 0.63.2
- `node` version: v14.18.3

### Relevant code or config:

N/A

### What you did:

Clone the repository locally and run the command `npm run test` or `npm test`.

### What happened:

The command fails with the following errors:

```
 FAIL  src/__tests__/to-contain-element.js
  ● Test suite failed to run

    TypeError: _interopRequireDefault is not a function

      at Object.<anonymous> (node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:3:17)

 FAIL  src/__tests__/to-be-disabled.js
  ● Test suite failed to run

    TypeError: _interopRequireDefault is not a function

      at Object.<anonymous> (node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:3:17)

 FAIL  src/__tests__/to-have-prop.js
  ● Test suite failed to run

    TypeError: _interopRequireDefault is not a function

      at Object.<anonymous> (node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:3:17)

 FAIL  src/__tests__/to-be-empty.js (6.679 s)
  ● Console

    console.error
      The above error occurred in the <AppContainer> component:
          in AppContainer (at src/index.js:26)

      Consider adding an error boundary to your tree to customize error handling behavior.
      Visit https://fb.me/react-error-boundaries to learn more about error boundaries.

      at console.Object.<anonymous>.console.error (node_modules/react-native/Libraries/LogBox/LogBox.js:33:14)

  ● .toBeEmpty

    TypeError: _interopRequireDefault is not a function

      at Object.<anonymous> (node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:3:17)

 FAIL  src/__tests__/to-have-style.js (7.117 s)
  ● .toHaveStyle › handles positive test cases

    TypeError: _interopRequireDefault is not a function

      at Object.<anonymous> (node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:3:17)

 FAIL  src/__tests__/to-have-text-content.js (7.122 s)
  ● .toHaveTextContent › handles positive test cases

    TypeError: _interopRequireDefault is not a function

      at Object.<anonymous> (node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js:3:17)
```

### Reproduction:

It's not really required to create a specific repository for reproducing the issue, as it can be reproduced with the current state of the `main` branch (https://github.com/testing-library/jest-native/commit/0dbda7191106f15a57eda1be10eadf95ad8bd81f).

### Problem description:

It's not expected that tests fail when using the `main` branch.

### Suggested solution:

I couldn't come up with a solution so far, further investigations should be performed in order to identify the cause of the error. I'd appreciate it if someone else could take a look at this or provide some guidance/clues.

### Can you help us fix this issue by submitting a pull request?

TBD

**Comment by TulipLi** [comment]
meet the same issue

**Comment by mdjastrzebski** [comment]
I confirm that this also happens to me. Seems like our setup stopped working. I wonder how that could slip through CI.

**Comment by mdjastrzebski** [comment]
Fixed by #78
